packages/wdio-config/src/shim.js

```diff
@@ -46,15 +46,9 @@ export let executeHooksWithArgs = async function executeHooksWithArgsShim (hooks
 }
 
 export let runTestInFiberContext = NOOP
-export let runFnInFiberContext = (fn, done) => {
+export let runFnInFiberContext = (fn) => {
     return function (...args) {
-        const result = fn.apply(this, args)
-
-        if (typeof done === 'function') {
-            return done(result)
-        }
-
-        return result
+        return Promise.resolve(fn.apply(this, args))
```

Comment:
Does this have any effect on our asynch / standalone users? Do we want to check and make sure that synch is running before removing promises that they might want be expecting?

Reply:
> Does this have any effect on our asynch / standalone users?

no, we have tests for custom commands now for sync (see smoke tests) and async (standalone) mode (see unit tests)